# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/no-profile.d-xprint.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We at least do not want the .csh file in /etc/profile.d - we probably need to
create an own one, but do not want to use the shipped one for bash.

  - Rene Rebe <rene@exactcode.de>

--- ./programs/Xserver/Xprint/etc/profile.d/Imakefile.orig	2004-08-25 19:27:47.950043112 +0200
+++ ./programs/Xserver/Xprint/etc/profile.d/Imakefile	2004-08-25 19:27:55.620876968 +0200
@@ -10,5 +10,3 @@
 
 all::
 
-InstallNamedProg(xprint.csh,xprint.csh,$(ETCDIR)/profile.d)
-InstallNamedProg(xprint.sh,xprint.sh,$(ETCDIR)/profile.d)
